isaacConfig.cmake

```diff
@@ -0,0 +1,141 @@
+# - Config file for the isaac package
+# It defines the following variables
+#  isaac_INCLUDE_DIRS - include directories for FooBar
+#  isaac_LIBRARIES    - libraries to link against
+#  isaac_DEFINITIONS  - necessary definitions
+#
+# It defines the following options
+#  ISAAC_THREADING
+#  ISAAC_SHOWBORDER
+#  ISAAC_CUDA
+#  ISAAC_ALPAKA
+#  ISAAC_JPEG
+
+###############################################################################
+# isaac
+###############################################################################
+cmake_minimum_required (VERSION 2.6)
+cmake_policy(SET CMP0048 OLD)
+project (isaac)
+
+set(isaac_INCLUDE_DIRS ${isaac_INCLUDE_DIRS} "${isaac_DIR}/insitu")
+set(isaac_INCLUDE_DIRS ${isaac_INCLUDE_DIRS} "${isaac_DIR}/insitu/isaac")
+
+set(isaac_DEFINITIONS ${isaac_DEFINITIONS} "-std=c++11")
```

Comment:
ah I see, because it is the list of definitions you give out to the actual targets... will still collide with other flags, e.g., someone is using already a `-std=c++14` flag in his definitions o.0